github/Repository.py

```diff
@@ -3166,6 +3166,7 @@ def get_issues(
         milestone: Milestone | Opt[str] = NotSet,
         state: Opt[str] = NotSet,
         assignee: NamedUser | Opt[str] = NotSet,
+        type: Opt[str] = NotSet,
```

Comment:
Given this is a positional argument, we have to add this at the end of the argument list, unfortunately. Otherwise this might be breaking for some users.